cmd/tldr/main.go

```diff
@@ -176,6 +202,9 @@ func main() {
 	random := flag.Bool("random", false, randomUsage)
 	flag.BoolVar(random, "r", false, randomUsage)
 
+	history := flag.Bool("history", false, historyUsage)
+	flag.BoolVar(history, "h", false, historyUsage)
```

Comment:
I think the shortcut for `h` isn't really good, when using a program `h` or `help` is used for the help output and I think `history` would be enough for that case, what do you think?

Reply:
good point, i have change to use 't' now, but still not too sure if it is good.